raiden-dapp/src/components/overlays/InfoOverlay.vue

```diff
@@ -38,60 +40,77 @@ export default class InfoOverlay extends Vue {
 </script>
 
 <style lang="scss" scoped>
+@import '@/scss/dimensions';
 @import '@/scss/mixins';
 @import '@/scss/colors';
 
 .info-overlay {
-  align-items: center;
   background-color: $transfer-screen-bg-color;
   border-radius: 10px;
-  display: flex;
-  flex-direction: column;
-  height: calc(844px);
-  padding: 0 46px;
+  height: 844px;
   position: absolute;
   width: 100%;
   z-index: 102;
+
   @include respond-to(handhelds) {
     border-radius: 0;
     height: 100vh;
+    overflow-y: auto;
   }
 
-  &__header {
+  &__content {
+    align-items: center;
     display: flex;
-    height: 80px;
-    width: 100%;
-  }
+    flex-direction: column;
+    height: 100%;
+    @include respond-to(handhelds) {
+      margin-top: $ios-safe-area;
```

Comment:
I think this is wrong. You set the height to 100% and now you push it down by a margin. I think `padding-top` is correct. Else you push the content down. If you go into the dev-tools and inspect the layout you can see this.

Reply:
You are right, padding would make more sense. But same comment as above. I will remove this completely anyways.

Reply:
Same as in the conversation above. 👍🏾